### ISSUE TITLE
Standard deviation calculation method in the paper

### ISSUE THREAD

**Issue by yuwxyun275** [HHW-zhou/TDHNN#5]
Thank you very much for your great contribution. Does the standard deviation reported in the paper refer to the last 20 epochs, or does it mean running 250epochs each time and averaging over 20 times?

**Comment by HHW-zhou** [comment]
Thank you for your attention. "20 runs" refers to repeating the training and testing process 20 times with the same hyperparameters unchanged, while only changing the random seed each time, to minimize the impact of randomness.

**Comment by yuwxyun275** [comment]
Is there no code to calculate this process?

**Comment by HHW-zhou** [comment]
It's quite simple. You can create a `seed_list` on line 18 of `main.py`, then iterate through this `seed_list` and place the content after line 18 inside the loop.

**Comment by yuwxyun275** [comment]
I really appreciate your quick response. However, when running the NTU dataset, the test loss value shows as null, but this does not affect the final result. Is this a normal phenomenon?

![Image](https://github.com/user-attachments/assets/f377708e-7c96-4734-9279-a719c57b1671)

**Comment by yuwxyun275** [comment]
"When running MVCNN on the NTU dataset, the accuracy shown is as in the image, which seems to be more than 10% higher than what is reported in the paper. A similar situation also occurred with NTU's GVCNN and GV+MV. What could be the reason for this?

![Image](https://github.com/user-attachments/assets/30013c2b-4035-496b-884a-fed6d1509352)

**Comment by HHW-zhou** [comment]
The occurrence of NaN in the loss seems to be due to the loss value being too small, falling below the lower bound of the data type, resulting in underflow. However, this should not be an issue during the inference phase. Regarding performance, we have reported relatively conservative experimental data; the actual performance of the model is somewhat better than what we have reported.

**Comment by yuwxyun275** [comment]
I am very grateful for your answer.

**Comment by yuwxyun275** [comment]
After setting the split_ratio to a negative number, it seems that the NTU results are more in line with the results in the paper. Is it possible that I misremembered the partition parameters used at that time?

**Comment by HHW-zhou** [comment]
> After setting the split_ratio to a negative number, it seems that the NTU results are more in line with the results in the paper. Is it possible that I misremembered the partition parameters used at that time?

Regarding this issue, please refer to: https://github.com/HHW-zhou/TDHNN/issues/4#issuecomment-2290754993
